api/pkg/apps/cms/types.go

```diff
@@ -63,22 +80,36 @@ type CaseTemplate struct {
 	FormElements []CaseTemplateFormElement `json:"formElements" bson:"formElements"`
 }
 
+func NewCaseTemplate(templateString string) (CaseTemplate, error) {
+	var ct CaseTemplate
+	if err := json.Unmarshal([]byte(templateString), &ct); err != nil {
+		return ct, err
+	}
+	return ct, nil
+}
+
```

Comment:
This is exactly the same signature as the vanilla json.Unmarshal, so this method is not needed
You can replace the calls by json.Unmarshal